1 HW_Standard/HW_Graveler.yaml

```diff
@@ -20,3 +20,26 @@ nature: Serious
 ivs: {hp: 31, atk: 31, def: 0, spA: 31, spD: 31, spe: 31}
 evs: {hp: 252, atk: 252, def: 0, spA: 252, spD: 252, spe: 0}
 hidden: false
+---
+species: Graveler
+ingamename: Wally Wall
+displayname: Nurikabe
+setname: Halloween
+tags: [halloween]
+item: Protector
+ability: Filter
+moves:
+    - [Iron Defense]
+    - [Toxic]
+    - [Body Slam]
+    - [Block]

```

Comment:
helpless vs gastly line but would wall any physical varients thereof if it got boosts. Also walled by steels.